com.civicrm.hrjobroles/hrjobroles.php

```diff
@@ -188,3 +188,84 @@ function hrjobroles_civicrm_queryObjects(&$queryObjects, $type) {
         $queryObjects[] = new CRM_Hrjobroles_BAO_Query();
     }
 }
+
+/**
+ * This extension deals with multi values fields and
+ * options lists in a peculiar way. Because of that,
+ * the CiviCRM export function isn't capable to
+ * properly retrieve all the related information about
+ * the Job Role. With this hook, we have a chance to
+ * fix this, by manipulating the results and fixing
+ * the fields data
+ *
+ *
+ * @param $exportTempTable
+ * @param $headerRows
+ * @param $sqlColumns
+ * @param $exportMode
+ */
+function hrjobroles_civicrm_export( $exportTempTable, $headerRows, $sqlColumns, $exportMode ) {
+
+    $splitItemsQuery = function($items) {
+        return "SELECT SUBSTRING_INDEX(SUBSTRING_INDEX($items, '|', n.n), '|', -1) value
+FROM (
```

Comment:
I think the returned value may require a better code styling and indentation .. for example instead of one big string you can separate lines into strings and concat them  something like this :

return "Select * From ...... " .
           "Where ....... " .
           "And ......";